pyBumpHunter/bumphunter_2dim.py

```diff
@@ -97,37 +102,54 @@ class BumpHunter2D():
                  tested.
                  Each python list as dimension (Nstep), with Nstep the number of scan step for a given width
                  (different for every value of width).
-                 
-        
+
+
         min_Pval_ar : Array containing the minimum p-values obtained for the data (indice=0) and and the pseudo-
                       data (indice>0).
-        
+
         min_loc_ar : Array containing the positions of the windows for which the minimum p-value has been found
                      for the data (indice=0) and pseudo-data (indice>0).
-        
+
         min_width_ar : Array containing the width of the windows for which the minimum p-value has been found for
                        the data (indice=0) and pseudo-data (indice>0).
-        
+
         signal_eval : Number of signal events evaluated form the last scan.
-        
+
         signal_min : Minimum number of signal events ones must inject in the data in order to reach the required
                      significance (might differ from signal_eval).
-        
+
         signal_ratio : Ratio signal_min/signal_exp (signal strength).
-        
+
         data_inject : Data obtained after injecting signal events in the backgound.
-        
+
         sigma_ar : Numpy array containing the significance values obtained at each step.
-    '''
-    
+    """
+
     # Initializer method
-    def __init__(self,rang=None,mode='excess',
-                 width_min=[1,1],width_max=None,width_step=[1,1],scan_step=[1,1],
-                 Npe=100,bins=[20,20],weights=None,Nworker=4,
-                 sigma_limit=5,str_min=0.5,str_step=0.25,
-                 str_scale='lin',
-                 signal_exp=None,flip_sig=True,seed=None, useSideBand=False):
-        
+    def __init__(
+        self,
+        rang=None,
+        mode="excess",
+        width_min=None,
+        width_max=None,
+        width_step=[1, 1],
```

Comment:
Don't use this arguments, there is only one constructed. Mutating it will mutate it for a lifetime. In general, I would suggest to use `None` as the arguments